client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -55,6 +56,10 @@ class ActivityLogItem extends Component {
 
 	confirmRewind = () => this.props.confirmRewind( this.props.activity.rewindId );
 
+	trackContentLinkClick = ( { target: { dataset } } ) => {
+		analytics.tracks.recordEvent( 'calypso_activitylog_item_click', dataset );
```

Comment:
`dataset` right now is a `DOMStringMap` in Firefox. I think we should try passing a plain object instead.